sdk/Cargo.toml

```diff
@@ -17,5 +17,9 @@ postcard = { version = "1.0.8", features = ["alloc"] }
 uuid = { version = "1.9.1", features = ["v4", "fast-rng"] }
 thiserror = "1.0.61"
 
+[dev-dependencies]
```

Comment:
The sdk should also re-export.

Reply:
updated in https://github.com/nexus-xyz/nexus-zkvm/commit/f3ee8a1abc89008cd08ad6f0f1dc3c87384e6f40